lib/obs_github_deployments.rb

```diff
@@ -6,6 +6,7 @@
 loader.setup
 
 module ObsGithubDeployments
+  require "octokit"
```

Comment:
please move it under the `loader.setup`. We don't have to scope `octokit` inside the module `ObsGithubDeployments`

Reply:
Done